recipe/build.sh

```diff
@@ -11,7 +11,7 @@ if [[ -n "$MACOSX_DEPLOYMENT_TARGET" ]]; then
 fi
 
 cmake ${CMAKE_ARGS} -DOVITO_BUILD_DOCUMENTATION=ON \
-      -DPython3_ROOT_DIR="${PREFIX}" \
+      -DPython3_ROOT_DIR="${BUILD_PREFIX}" \
```

Comment:
This can be a bit critical - are you sure the path is correctly updated using `patchelf`? To my knowledge `patchelf` only replaces everything in `${PREFIX}` with the path where the user installed conda. So it might be reasonable to add a test to see if the python functionality is accessible in the final executable.

Reply:
Python scripting is only available in the "pro" version of ovito, as opposed to the "basic" version (https://www.ovito.org/about/ovito-pro) and for conda-forge, we only build the "basic" version.
Here, python is used to build the doc using sphinx and if the build doesn't build correctly, the build will fail.